### ISSUE TITLE
Still working ?

### ISSUE THREAD

**Issue by adharna** [GustavoMilitao/MageBot#1]
Hi

[17:00:13] (Connexion) Connexion
[17:00:13] (Connexion) Connected: 213.248.126.40
[17:00:13] (Debug) [SND] 182 (BasicPingMessage)
[17:00:16] (Debug) [RCV] 3 (HelloConnectMessage)
[17:00:17] (Debug) [SND] 4 (IdentificationMessage)
[17:00:17] (Connexion) Identification en cours.

**Comment by GustavoMilitao** [comment]
Then, it crashed?

**Comment by GustavoMilitao** [comment]
Sorry, last time I code MageBot was 2~3 months ago, and it was working fine.

**Comment by adharna** [comment]
@GustavoMilitao No, he stuck on [17:00:17] (Connexion) Identification en cours.

**Comment by GustavoMilitao** [comment]
It could be the version of dofus (it is declared in app.config). If isn`t that, I have to debug to find the bug, but I`m busy right now, maybe in two or more weeks. I hope I`ll can help you.

**Comment by adharna** [comment]
Yeah i try but no success

<?xml version="1.0"?>
<configuration>
    <startup> 
        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.0"/>
    </startup>
    <appSettings>
      <add key="Major" value="2" />
      <add key="Minor" value="42" />
      <add key="Release" value="1" />
      <add key="Revision" value="121975" />
      <add key="Patch" value="1" />
      <add key="BuildType" value="0" />
      <add key="Install" value="1" />
      <add key="Technology" value="1" />
      <add key="Lang" value="fr" />
      <add key="ServerID" value="0" />
      <add key="AutoConnect" value="true" />
      <add key="UseCertificate" value="false" />
      <add key="UseLoginToken" value="false" />
      <add key="SessionOptionalSalt" value="0" />
      <add key="Ip1" value="213.248.126.40"/>
      <add key="Port1" value="5555"/>
    </appSettings>
</configuration>
